### PR TITLE
Jira V3 - Update docs

### PR DESCRIPTION
**Pull request by anas-yousef**
## Status
- [ ] In Progress
- [X] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
Updated the documentation on how to acquire the **Cloud ID** when configuring Jira Cloud.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/Jira/Integrations/JiraV3/JiraV3_description.md [hunk 1]
```diff
@@ -34,10 +33,10 @@ The integration uses the *offline_access* scope, in order to retrieve refresh to
 * read:jira-work
 * read:jira-user
 * write:jira-work
-* read:jql:jira
 
 #### Granular Scopes
 
+* read:jql:jira
```
Comment (ShacharKidor) [inline Packs/Jira/Integrations/JiraV3/JiraV3_description.md:39]: please add this change to the readme file as well.

**Review by ShacharKidor — APPROVED** [review]
Nice!

File: Packs/Jira/Integrations/JiraV3/JiraV3_description.md [hunk 2]
```diff
@@ -21,9 +21,8 @@ Go to your [Developer console](https://developer.atlassian.com/console/myapps/)
 
 #### Cloud ID
 
-Go to your [Admin page](https://admin.atlassian.com/), and choose the `Jira Software` product by clicking the three dots and choosing **Manage users**. Your Cloud ID will appear in the URL:
+Go to your [Admin page](https://admin.atlassian.com/), click on the `Products` tab on the top banner, choose the appropriate site under `Sites and Products` on the left side bar. Your Cloud ID will appear in the URL:
```
Comment (ShirleyDenkberg) [inline Packs/Jira/Integrations/JiraV3/JiraV3_description.md:24]:
```suggestion
Go to your [Admin page](https://admin.atlassian.com/), click the **Products** tab on the top banner and choose the appropriate site under **Sites and Products** on the left side bar. Your Cloud ID will appear in the URL:
```

File: Packs/Jira/Integrations/JiraV3/README.md [hunk 3]
```diff
@@ -58,9 +58,8 @@ Go to your [Developer console](https://developer.atlassian.com/console/myapps/)
 
 #### Cloud ID
 
-Go to your [Admin page](https://admin.atlassian.com/), and choose the `Jira Software` product by clicking the three dots and choosing **Manage users**. Your Cloud ID will appear in the URL:
-`https://admin.atlassian.com/s/{Cloud_id}/users`
-(If you do not have it, click **Products** and add the `Jira Software` product to your products list.)
+Go to your [Admin page](https://admin.atlassian.com/), click on the `Products` tab on the top banner, choose the appropriate site under `Sites and Products` on the left side bar. Your Cloud ID will appear in the URL:
```
Comment (ShirleyDenkberg) [inline Packs/Jira/Integrations/JiraV3/README.md:61]:
```suggestion
Go to your [Admin page](https://admin.atlassian.com/), click the **Products** tab on the top banner and choose the appropriate site under **Sites and Products** on the left side bar. Your Cloud ID will appear in the URL:
```

File: Packs/Jira/ReleaseNotes/3_0_9.md [hunk 4]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### Atlassian Jira v3
+
+- Updated the documentation on how to acquire the **Cloud ID** when configuring Jira Cloud.
```
Comment (ShirleyDenkberg) [inline Packs/Jira/ReleaseNotes/3_0_9.md:6]:
```suggestion
Updated the documentation on how to acquire the **Cloud ID** when configuring Jira Cloud.
```

**Comment by ShirleyDenkberg** [comment]
@ShacharKidor Doc review completed.

**Pull request merged**